### ISSUE TITLE
"MUST follow accessible distribution practices" perhaps weakly defined?

### ISSUE THREAD

**Issue by rdeltour** [w3c/epub-specs#908]
One of the conformance requirements of [Discovery-enabled EPUB Publications](https://rawgit.com/IDPF/epub-revision/master/build/a11y/accessibility.html#sec-conf-discovery) is 

> Authors must follow accessible distribution practices, as described in [Distribution](https://rawgit.com/IDPF/epub-revision/master/build/a11y/accessibility.html#sec-distribution)

[Distribution](https://rawgit.com/IDPF/epub-revision/master/build/a11y/accessibility.html#sec-distribution) in turn contains only one normative requirement which is 

> Authors must take all steps within their power to ensure that distribution of their content does not negatively affect its discoverability or usability.

Perhaps this is too vague to be a MUST (how do we define "all steps within their power")? Maybe we can consider using a SHOULD instead.

**Comment by mattgarrish** [comment]
Yes, it's the weak link in the chain of specification + techniques. We can't control for every distribution channel, but we know that issues often arise in distribution that can scuttle anything in this specification.

We also have to normatively require people do something in the specification in order to have techniques that need to be followed.

A should is maybe more realistic, but it doesn't solve the problem. Perhaps we should call out the two practices we have in the techniques, and leave the wishy-washy language to cover other things we aren't aware of, or that might spring to life in the techniques between revisions?

**Comment by mattgarrish** [comment]
It was decided on the call today to decouple distribution from the categories of publication, as you cannot test or certify content accessible if it is dependent on being distributed in a certain way (i.e., how can you verify until after it has been distributed, in which case it's too late).

Distribution is a separate concern from authoring, so will be retained as separate guidance.
